engine/runtime/src/main/scala/org/enso/compiler/core/R.java

```diff
@@ -0,0 +1,5 @@
+package org.enso.compiler.core;
+
+// Just to demonstrate we can now compile records
+public record R (String name) {
```

Comment:
We have to remove this before integration, but I can confirm that:
```bash
enso$ sbt --java-home ~/bin/graalvm-ce-java11-21.3.0/ runtime/compile
enso$ find . | grep /R.class
./engine/runtime/target/scala-2.13/classes/org/enso/compiler/core/R.class
./engine/runtime/target/scala-2.13/classes/META-INF/versions/16/org/enso/compiler/core/R.class
```
which clearly demonstrates the compilation is working!

PS: I had to perform complete cleanup `rm -rf *; git checkout -f .` to get the compilation working, but now everything works.

Reply:
I've hit the problem 
```
Bad service configuration file, or exception throw
while constructing Processor object: javax.annotation.processing.Processor: 
Provider org.enso.interpreter.dsl.MethodProcessor
```
again (after switching back to `develop` branch). It was enough to
```bash
enso$ rm ./lib/scala/interpreter-dsl/target/scala-2.13/classes/META-INF/services/javax.annotation.processing.Processor
```

Hypothesis: Scala's incremental compilation detects classes are out of date and removes just `.class` files, but not the `META-INF/services` registration. Next invocation of JavaC then finds the registration, but fails the load the registered class.

Reply:
yes, the problem is independent of that change. It's been like that before.

Reply:
Btw `sbt clean` should do it (at least it did in my case)